docs/docs/documentation/tutorials/custom-agent.md

```diff
@@ -48,8 +48,176 @@ Run the following command to create a new custom agent:
 sam add agent --gui
 ```
 
+:::tip
+You can create an agent either by using the `sam add agent` command or by creating a new plugin of type agent, `sam plugin create my-hello-agent --type agent`. 
+
+Check the [`Agent or Plugin, Which to use?`](../concepts/plugins.md#agent-or-plugin-which-to-use) guide to decide which is the better choice for you.
```

Comment:
For information and recommendations about these options, see [`Agent or Plugin; Which to Use?`](../concepts/plugins.md#agent-or-plugin-which-to-use) .